packages/core/src/components/Timeline/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export type TimelineProps = {
+  /** Collection of timeline items. */
+  children: NonNullable<React.ReactNode>;
+};
+
+/** A chronologically ordered list of items. Ordering is determined by caller. */
+export default function Timeline({ children }: TimelineProps) {
+  return <>{children}</>;
```

Comment:
this wrapping tag should be `<ol>` - this is an ordered list of items (by time). the <Item> component should be wrapped with `<li>` tags

Reply:
That's a good idea